src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+}
+  
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  width: 100%;
+  padding: 0 50px;
+}
+  
+.nav_list {
+  display: flex;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+}
+
+.links {
+  margin-right: 20px;
+  padding: 23px 0;
```

Comment:
Set this paddings to the link instead of `li`

Reply:
I did it earlier, however, in this case all the paddings worked wrongly, they did not affect the links at all